src/Mafia/Home.hs

```diff
@@ -62,6 +62,7 @@ installBinary package deps = do
   -- We can remove this at some point
   -- NOTE: This makes no attempt at being threadsafe
   whenM (liftIO (doesFileExist prefix)) (removeFile prefix)
+  createDirectoryIfMissing True prefix
```

Comment:
it doesn't throw if the dir already exists http://hackage.haskell.org/package/directory-1.2.5.0/docs/src/System.Directory.html#createDirectoryIfMissing